extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+// Name: Twitch
+// ID: twitch
+// Description: Blocks that allow you to stream easier on TurboWarp.
+// By: BOP <https://scratch.mit.edu/users/CEFSharp/>
+
+(function (Scratch) {
+  "use strict";
+
+  let chat = null;
+  let token = null;
+  let text = null;
+  let image = null;
+
+  class Twitch {
+    getInfo() {
+      return {
+        id: "twitch",
```

Comment:
Please put your username in the extension id; something like "bopTwitch". Do it however you like, but it must include some form of your username.

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/9653b524-6047-4791-8190-93ad8174be84)